bar-trender-front-end/src/components/ApiFormSubmit.js

```diff
@@ -28,51 +32,26 @@ class POSTForm extends Component {
         this.setState({ [e.target.name]: e.target.value });
     }
 
-    async handleSearch(e) {
+    async handleSearch() {
         // this.props.searchEngine(this.state.term);
-
-
-        // console.log(this.state.filters);
-        // let temp_state = JSON.stringify(this.state.filters)
-
-        // console.log(this.state)
-
-        let beers = [];
-        let zones = [];
-
-        if (this.state['Paulaner'] == "on") {
-            beers.push("Paulaner");
-        }
-        if (this.state['Cruzcampo'] == "on") {
-            beers.push("Cruzcampo");
-        }
-        if (this.state['Alameda'] == "on") {
-            zones.push("Alameda");
-        }
-        if (this.state['Triana'] == "on") {
-            zones.push("Triana");
-        }
-
-        let params = "";
-        beers.map((e, i) => {
-            if (i === 0) {
-                params += "beers=" + e;
-            } else {
-                params += "," + e;
-            }
-        })
-        zones.map((e, i) => {
-            if (i === 0) {
-                params += "&zones=" + e;
-            } else {
-                params += "," + e;
+        
+        //alert("A form was submitted: " + JSON.stringify(this.state));
+        this.state = {
+            "filters":{
             }
-        })
-
-        let data = this.state.data;
-        // this.context.router.push("/list");
-        window.location.href = 'list';
-
+        };
+        await fetch('http://localhost:8000/v1/search/establishments', {
+            method: 'POST',
+            // We convert the React state to JSON and send it as the POST body
+            body: JSON.stringify(this.state),
+            headers: {
+                'Content-Type': 'application/json'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7145559)
```suggestion
                "Content-Type": 'application/json'
```